### ISSUE TITLE
error creating postgres store in geonode geoserver

### ISSUE THREAD

**Issue by NyakudyaA** [GeoNode/geonode#2661]
I have build my container but when I try to create a store in geoserver I get an error:

![error3](https://cloud.githubusercontent.com/assets/2510900/18986676/377cae1c-86fe-11e6-9f88-401ae23b374f.png)

My confirguration is like below:
![error1](https://cloud.githubusercontent.com/assets/2510900/18986694/4adb7290-86fe-11e6-96bc-f51a049fcaa0.png)

I can exec into the postgres container and i notice that extension postgis is not installed.
![error](https://cloud.githubusercontent.com/assets/2510900/18986723/6a56d722-86fe-11e6-86dd-7f9362d9107c.png)

**Comment by simod** [comment]
Hi, seems that you are missing postgis in your container. You should install it.

**Comment by NyakudyaA** [comment]
I am just using the default postgres image. So should we then modify the postgres image

**Comment by francbartoli** [comment]
@NyakudyaA It is not expected to create a new postgis datastore for the geonode platform. What are you trying to achieve here?

**Comment by NyakudyaA** [comment]
I had big datasets (shapefile) which I needed to directly upload to my geonode database store. After this I needed to publish the data in geoserver and then update my geonode with geoserver. The python manage.py upload layers only works with shapefiles

**Comment by francbartoli** [comment]
Have you considered the option to load shapefiles into the data volume and then run importlayers [http://docs.geonode.org/en/master/tutorials/admin/admin_mgmt_commands/](http://docs.geonode.org/en/master/tutorials/admin/admin_mgmt_commands/)?

**Comment by NyakudyaA** [comment]
yes I have considered running those commands. It is just that by having the data in the database it opens up a whole lot of possibilities and also creating views in the database and geoserver sql views. I would prefer to upload the data into the db. Unless that is the only solution to use shapefiles
